include/openssl/kdf.h

```diff
@@ -79,6 +79,13 @@ void EVP_KDF_names_do_all(const EVP_KDF *kdf,
 # define EVP_KDF_CTRL_SET_UKM                0x16 /* unsigned char *, size_t */
 # define EVP_KDF_CTRL_SET_CEK_ALG            0x17 /* char * */
 # define EVP_KDF_CTRL_SET_SHARED_INFO        EVP_KDF_CTRL_SET_SSKDF_INFO
+# define EVP_KDF_CTRL_SET_ARGON2_LANES       0xf1 /* uint32 */
```

Comment:
Unless there is a good reason, I think these CTRL's should not be needed (The controls were there for the old API's, OSSL_PARAMS should be sufficient for any new KDF's).